plugins/wls.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: wls
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/mohatb/kubectl-wls
+  shortDescription: Windows-linux-shell allows you to start a shell on windows or linux nodes
```

Comment:
`Windows-linux-shell allows you to ` is redundant, makes "short" description too long.